README.md

```diff
@@ -6,109 +6,87 @@
-    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 642AC823
-    sudo apt-get update
-    sudo apt-get install sbt
-    ```
-1. Install Verilator.
-    We currently recommend Verilator version 4.006.
-    Follow these instructions to compile it from source.
-
-    1. Install prerequisites (if not installed already):
-        ```
-        sudo apt-get install git make autoconf g++ flex bison
-        ```
-
-    2. Clone the Verilator repository:
-        ```
-        git clone http://git.veripool.org/git/verilator
-        ```
-
-    3. In the Verilator repository directory, check out a known good version:
-        ```
-        git pull
-        git checkout verilator_4_006
-        ```
-
-    4. In the Verilator repository directory, build and install:
-        ```
-        unset VERILATOR_ROOT # For bash, unsetenv for csh
-        autoconf # Create ./configure script
-        ./configure
-        make
-        sudo make install
-        ```
-
-### Arch Linux
+# Build circuits with Chisel
 
-```
-yaourt -S firrtl-git verilator sbt
-```
+When you are ready to build and test your own circuits. Chisel3 uses the Sonatype/Nexus/Maven package management systems to seamlessly deliver the environment and extension
+libraries. Set up is easy. The chisel-template repo provides a setup and build environment.
 
-### Windows
+## [Take me to the Chisel-Template](https://github.com/freechipsproject/chisel-template)
 
-[Download and install sbt for Windows](https://www.scala-sbt.org/download.html).
+For more a more complex template that includes rocket-chip and more, visit the [rebar repository](https://github.com/ucb-bar/project-template).
+To learn more about installing Chisel3 visit [The wiki installation page](Installation)
 
-#### Simulation on Windows
+---
 
-The chisel3 regression tests use Verilator as the simulator, but Verilator does not work well on Windows natively.
-However, Verilator works in [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) or in other Linux-compatible environments like Cygwin.
+# Documentation and other Resources
 
-Alternatively, if you're using [PeekPokeTester](https://github.com/freechipsproject/chisel-testers) or the [Testers2 alpha](https://github.com/ucb-bar/chisel-testers2), you can use [treadle](https://github.com/freechipsproject/treadle) as the simulation engine.
-Treadle is a FIRRTL simulator written in Scala, and works on any platform that can run Scala code.
-It can simulate any pure Chisel design, but cannot simulate Verilog code and hence will not work on BlackBoxes / ExtModules which do not have corresponding greybox definitions.
+## [Chisel3 Wiki](/freechipsproject/chisel3/wiki)
 
-There are no issues with generating Verilog from Chisel, which can be pushed to FPGA or ASIC tools.
+## [Chisel3 API Documentation](https://chisel.eecs.berkeley.edu/api/latest/chisel3/index.html)
 
-### Mac OS X
+## [Chisel3 Cheat Sheet](https://chisel.eecs.berkeley.edu/doc/chisel-cheatsheet3.pdf)
```

Comment:
I'd argue the cheatsheet should go first. It's more curated (though perhaps somewhat out of date) compared to the API docs, which I think highly varies in quality.

Reply:
I would actually say wiki -> cheatsheet -> API

Reply:
I think it depends on the degree of completeness and curation. I don't know much about how complete the wiki is, but the cheatsheet is intended to be a fairly complete and uniformly polished (though brief) overview of the APIs, and I think there's serious value in that compared to a more detailed but also sparse documentation. In any case, they're both complementary, and ordering might be less important than a short blurb about what each resource is intended to be used for.